weave-js/src/components/Panel2/useAssetFromArtifact.ts

```diff
@@ -155,16 +155,19 @@ export const useSignedUrlWithExpiration = (
         }
 
         // eslint-disable-next-line wandb/no-unprefixed-urls
-        const response = await fetch(directUrl.result + '?redirect=false', {
-          credentials: 'include',
-          method: 'GET',
-          mode: 'cors',
-          // TODO(np): This can and should be sent via cookie, but it's not
-          // being correctly set, breaking this code in devprod and integration tests.
-          headers: {
-            'use-admin-privileges': 'true',
-          },
-        });
+        const response = await fetch(
+          directUrl.result + '?redirect=false&content-disposition=inline',
```

Comment:
I feel like setting `content-disposition` to `inline` for every direct URL would be fine, but I'm not sure. Will manually test with different panel types once [the gorilla change](https://github.com/wandb/core/pull/16215) is deployed.